src/libraries/Common/src/Interop/Unix/System.Native/Interop.GetDomainName.cs

```diff
@@ -33,7 +33,7 @@ internal static unsafe string GetDomainName()
             }
 
             // Marshal.PtrToStringAnsi uses UTF8 on Unix.
-            return Marshal.PtrToStringAnsi((IntPtr)name);
+            return Marshal.PtrToStringAnsi((IntPtr)name) ?? string.Empty;
```

Comment:
I saw @stephentoub's comment about such change. It looks more safe.

Reply:
I'm not sure which comment you're referring to, but in this case, the right answer is to use `!` rather than add `?? string.Empty`.  This PtrToStringAnsi call will not return null in this case.

Reply:
Fixed.